src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -81,7 +81,7 @@ object RpmPlugin extends AutoPlugin {
     rpmConflicts := Seq.empty,
     rpmSetarch := None,
     rpmChangelogFile := None,
-    rpmBrpJavaRepackJars := false,
+    rpmBrpJavaRepackJars := true,
```

Comment:
Together with the fix  this doesn't change the default behaviour, right?

I'm not building rpm's on a daily basis, so I'm not sure what impact this has. When we introduced
this it was because `rpm:packageBin` took a long time to extract and repack all the jar files, which
is IMHO wasted time. I would prefer `false` as a default.